vnext/Shared/OInstance.cpp

```diff
@@ -747,12 +748,17 @@ std::vector<std::unique_ptr<NativeModule>> InstanceImpl::GetDefaultNativeModules
   modules.push_back(std::make_unique<CxxNativeModule>(
       m_innerInstance,
       "ExceptionsManager",
-      [&]() -> std::unique_ptr<xplat::module::CxxModule> {
```

Comment:
This seemed super sketch, since we implicitly capture the this pointer and use it async. Moved to value semantics which do effectively the same thing.